<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>X100 control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresource.qrc">
    <normaloff>:/img/img/Rvbust.png</normaloff>:/img/img/Rvbust.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <widget class="QLabel" name="loop">
      <property name="text">
       <string>loop</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="imgs_Edit">
      <property name="text">
       <string>1,2,3,4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="step">
      <property name="text">
       <string>step</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QPushButton" name="Capture_cmd">
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="num">
      <property name="text">
       <string>num</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QSpinBox" name="loop_spinBox">
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QSpinBox" name="step_spinBox">
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="imgs">
      <property name="text">
       <string>Capture CMD: imgs</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="num_spinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="9">
     <layout class="QVBoxLayout" name="consoleLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_2"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionClear"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="controlBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionLED_ON"/>
   <addaction name="actionLED_OFF"/>
   <addaction name="separator"/>
   <addaction name="actionStepperMotor0_up"/>
   <addaction name="actionStepperMotor0_down"/>
   <addaction name="separator"/>
   <addaction name="actionStepperMotor2_up"/>
   <addaction name="actionStepperMotor2_down"/>
   <addaction name="separator"/>
   <addaction name="actionCapture"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/connect.png</normaloff>:/img/img/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/disconnect.png</normaloff>:/img/img/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/application-exit.png</normaloff>:/img/img/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/settings.png</normaloff>:/img/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/clear.png</normaloff>:/img/img/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionLED_ON">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/led-ON.png</normaloff>:/img/img/led-ON.png</iconset>
   </property>
   <property name="text">
    <string>LED_ON</string>
   </property>
   <property name="toolTip">
    <string>LED_ON</string>
   </property>
  </action>
  <action name="actionLED_OFF">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/led-off.png</normaloff>:/img/img/led-off.png</iconset>
   </property>
   <property name="text">
    <string>LED_OFF</string>
   </property>
   <property name="toolTip">
    <string>LED_OFF</string>
   </property>
  </action>
  <action name="actionStepperMotor0_up">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/motor.png</normaloff>:/img/img/motor.png</iconset>
   </property>
   <property name="text">
    <string>StepperMotor0_up</string>
   </property>
   <property name="toolTip">
    <string>StepperMotor0_up</string>
   </property>
  </action>
  <action name="actionStepperMotor0_down">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/motor.png</normaloff>:/img/img/motor.png</iconset>
   </property>
   <property name="text">
    <string>StepperMotor0_down</string>
   </property>
   <property name="toolTip">
    <string>StepperMotor0_down</string>
   </property>
  </action>
  <action name="actionStepperMotor2_up">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/motor.png</normaloff>:/img/img/motor.png</iconset>
   </property>
   <property name="text">
    <string>StepperMotor2_up</string>
   </property>
   <property name="toolTip">
    <string>StepperMotor2_up</string>
   </property>
  </action>
  <action name="actionStepperMotor2_down">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/motor.png</normaloff>:/img/img/motor.png</iconset>
   </property>
   <property name="text">
    <string>StepperMotor2_down</string>
   </property>
   <property name="toolTip">
    <string>StepperMotor2_down</string>
   </property>
  </action>
  <action name="actionCapture">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/Capture.png</normaloff>:/img/img/Capture.png</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="toolTip">
    <string>Capture</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/stop.png</normaloff>:/img/img/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresource.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>Button()</slot>
 </slots>
</ui>
